src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -868,6 +884,14 @@ static DistributeObjectOps Type_Rename = {
 	.address = RenameTypeStmtObjectAddress,
 	.markDistributed = false,
 };
+static DistributeObjectOps View_Rename = {
```

Comment:
maybe add some comments along the lines:
/* PG supports ALTER TABLE on a view as well.
* If user types ALTER VIEW .. RENAME, go through this View_Rename, 
* if users types ALTER TABLE .. RENAME, initially goes through Rename_ANY, which then forwards command to the functions of  View_Rename.
*/


(for other ops as well, because it is really non-trivial to understand this. I wish PG has not allowed this :) )